### ISSUE TITLE
[Bug] Blank ad placeholders when scrolling homepage.

### ISSUE THREAD

**Issue by DesperatePea** [arichornlover/uYouEnhanced#622]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the Wiki?

- [X] Yes, I have read the Wiki

### iOS/iPadOS version

Ios 17.4.1

### Device model

iPhone 13

### Sideload method

AltStore

### Tweak version

uYouEnhanced_19.18.2-3.0.3.ipa

### Do you use YouTube Premium?

❌ No, I don't use YouTube Premium

### Describe the bug

When opening the home screen, there are big blank boxes in-between the youtube thumbnails. I think its trying to load an ad, but it gets blocked. Makes it very annoying to scroll.

### Steps to reproduce the issue

1. Open uYou
2. Scroll trough homepage.
3. Blank boxes appear as you  scroll, making thumbnails pop in and out of the list.

### Are you using the newest version of uYouEnhanced?

✅ Yes, I'm using the latest version

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by DesperatePea** [comment]
Nah, i think i will just deal with it. The point of using this is to bypass ads. I hope a proper fix comes along eventually. Thanks for the answer!

**Comment by DesperatePea** [comment]
I noticed the problem appears on my subscribers homepage as well. Homepage + subscribers list.

**Comment by The-Hokage1337** [comment]
happening to me too homepage one seems fixed but it happens on subs page now

**Comment by DesperatePea** [comment]
I just saw the same error on the "my likes" list as well. It basically cuts off the top half of my liked videos list, so this one is quite annoying. 
![Skjermbilde 2024–05–23 kl  18 22 45](https://github.com/arichornlover/uYouEnhanced/assets/73502428/832689ed-67da-47a1-81d3-ca652be8fd51)

**Comment by Invisible40** [comment]
> I just saw the same error on the "my likes" list as well. It basically cuts off the top half of my liked videos list, so this one is quite annoying. ![Skjermbilde 2024–05–23 kl  18 22 45](https://private-user-images.githubusercontent.com/73502428/333266989-832689ed-67da-47a1-81d3-ca652be8fd51.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4NTA2MjcsIm5iZiI6MTcxODg1MDMyNywicGF0aCI6Ii83MzUwMjQyOC8zMzMyNjY5ODktODMyNjg5ZWQtNjdkYS00N2ExLTgxZDMtY2E2NTJiZThmZDUxLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMFQwMjI1MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02Mzk3ZDlhMTNiZTRjNDAyZWNjNzNiMTgyNDAzNDgzZDUxOWNiYzZjNWUyNGM3Yzc0Mzc1NTcwNzM1ZWYzZmIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.aQ5o7jAZcHYQepW0Vuk0cK4KodynAwrJVzw-ai5Ql78)

To DesperatePea - [this ](https://github.com/arichornlover/uYouEnhanced/issues/658 )solved the issue
